flexget/tests/test_sftp_download.py

```diff
@@ -14,63 +14,57 @@ class TestSftpDownload:
     _config = """
         templates:
           anchors:
-            _sftp_list: &base_sftp_download
-                to: {{ download_path }}
-                socket_timeout_sec: 2
-                connection_tries: 1
+            _sftp_download: &base_sftp_download
```

Comment:
Smart use of anchors to reduce repetition. If you wanted to get really fancy, I think anchors expand recursively, but this is probably a good balance of DRYness and simplicity.